production/kubernetes/agent-bare.yaml

```diff
@@ -61,22 +76,33 @@ spec:
         - -config.file=/etc/agent/agent.yaml
         command:
         - /bin/agent
-        env:
-        - name: HOSTNAME
-          valueFrom:
-            fieldRef:
-              fieldPath: spec.nodeName
         image: grafana/agent:v0.23.0
         imagePullPolicy: IfNotPresent
-        name: agent
+        name: grafana-agent
         ports:
-        - containerPort: 12345
+        - containerPort: 80
```

Comment:
Does 12345 -> 80 line up with the configs we'll tell people to deploy?

Reply:
nope, reverting